docs/proposals/InputColor/README.md

```diff
@@ -0,0 +1,66 @@
+| ------------- | ------------------------------------- | ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `value`       | `string`                              | `#7db00e`        | This will be the hex color that will be displayed                                                                                                                                                                                                                                                                                                                    |
+| `placeholder` | `string`                              | `Choose a color` | This will be used to give context on where/what the color will be used for                                                                                                                                                                                                                                                                                           |
+| `onChange`    | `onChange(newValue: colorName): void` | none             | Callback that fires when the color is changed changed                                                                                                                                                                                                                                                                                                                |
+| `dataListId`  | `string?`                             | none             | This will allow us to link a color datalist                                                                                                                                                                                                                                                                                                                          |
+| `dataList`    | `string[]?`                           | none             | This will allow us to pass in some default colors to be listed when the color picker is selected but also allow them to access the color wheel. We currently are using this for the color picker right now. the datalist will not be another component however. This is a standard HTML element(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist). |
```

Comment:
```suggestion
| `colorSuggestions`    | `Color[]?`                           | none             | This will allow us to pass in some default colors to be listed when the color picker is selected but also allow them to access the color wheel. We currently are using this for the color picker right now. the datalist will not be another component however. This is a standard HTML element(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist). |
```

We probably want to give the colors in this list names? Making an interface for them might help with that.

eg.

```ts
interface Color {
  name: string;
  value: string;
}

const red = {
  name: "Red",
  value: "#ff0000",
}
```

Reply:
Talked to the designer and I think we will just let the datalist vanish when we swap to the atlantis component since it doesn't work across all browsers. This should clear out the last of the uncertainty hopefully.